README.md

```diff
@@ -280,6 +280,49 @@ Please go the Remote Management tab of the Settings page and check which remote
 
 ![Remote Management Tab](./assets/screenshot-3.png)
```

Comment:
I will create a new screenshot for this. As it will be for my localhost it really shouldn't matter that it's published.